core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.controller.ts

```diff
@@ -100,6 +146,94 @@ angular.module('oppia').controller('TranslationModalController', [
       $scope.activeWrittenTranslation.html = '';
     };
 
+    $scope.getElementTexts = function(elements, type) {
```

Comment:
Can you move the new functions below `suggestTranslatedText` and then order them in order in which they are called? This makes it easier to understand the flow.